### PR TITLE
Added: ago(1), a command to compute days between dates.

### PR DESCRIPTION
**Pull request by lindig**
This is a small utility in pure OCaml with no external dependencies.
I hope it be useful but it also serves as my personal testbed to move
bigger existing project to OPAM.

### CODE REVIEW

File: packages/ago/ago.0.1/descr [hunk 1]
```diff
@@ -0,0 +1,7 @@
+ago - compute number of days between two calendar dates
+
+Ago computes the difference in days between two calendar dates provided
+as arguments. If just one date is given, the the current date is taken as
```
Comment (yallop) [inline packages/ago/ago.0.1/descr]: "the the"

**Comment by lindig** [comment]
Fixed the typo and pushed an update. Do I need to create a new pull request?

**Comment by yallop** [comment]
No need for a new request: pushing updates to the branch is fine.

**Comment by yallop** [comment]
It [looks](https://travis-ci.org/ocaml/opam-repository/jobs/45728213) like you need a version constraint:

```
# File "agolex.mll", line 20, characters 30-32:
# Error: Unbound value |>
```

Something like the following should do the trick:
```
available: [ ocaml-version >= "4.01.0" ]
```

**Comment by yallop** [comment]
Thanks!

**Pull request merged**